typeDefs.js

```diff
@@ -278,9 +278,9 @@ const typeDefs = gql`
     updateNews(id: ID!, news: NewsInput!): NewsResult
 
     "User Mutation"
-    registerUser(user: userRegisterInput!): User
-    loginUser(user: userLoginInput!): User
-    loginAdmin(user: UserInput!): User
+    registerUser(user: UserInput!, language: Int!): User
+    loginUser(loginInput: userLoginInput!): User
```

Comment:
Please, rename `userLoginInput` to `LoginInput`, there is no need to include word `user`

Reply:
Renamed!